graphrag/vector_stores/lancedb.py

```diff
@@ -16,6 +15,7 @@
     VectorStoreDocument,
     VectorStoreSearchResult,
 )
+import lancedb as lancedb
```

Comment:
Minor comment...I don't understand this import. Wouldn't `import lancedb` do the same thing?

Reply:
That's the import issue we have been facing and reporting in the chat

Reply:
But what Josh means is the as lancedb